### ISSUE TITLE
Alt text missing using WP Bakery

### ISSUE THREAD

**Issue by fredperes** [humanmade/network-media-library#80]
I'm trying to insert alt text with no luck. The website uses Salient theme and WPBakery. I insert the alt text in the Media section, but is not displayed in the source code:

![Captura de Tela 2021-01-12 às 14 16 48](https://user-images.githubusercontent.com/12846199/104348780-e5d0b580-54e0-11eb-8ac8-69229a4a8238.jpg)

If I deactivate the plugin, the alt text is displayed.

**Comment by locke85** [comment]
Same [issue](https://github.com/login?return_to=https%3A%2F%2Fgithub.com%2Fhumanmade%2Fnetwork-media-library) here. Is there a fix available to the meantime? If not, what would be a work around in the interim?
